src/db/db.js

```diff
@@ -0,0 +1,20 @@
+import { Pool } from 'pg';
+import env from '../config/envConf';
+
+const pool = new Pool({
+  connectionString: env.DATABASE_URL,
```

Comment:
I can't seem to locate `env.DATABASE_URL` in your `envConf` file. Please clarify.

Reply:
I have explicitly added the variable in #32